src/Skybrud.Umbraco.Redirects/Controllers/Api/RedirectsController.cs

```diff
@@ -61,13 +67,15 @@ public class RedirectsController : UmbracoAuthorizedApiController {
         [HttpGet]
         public ActionResult GetRootNodes() {
 
-            RedirectRootNode[] rootNodes = _redirects.GetRootNodes();
-
-            return new JsonResult(new {
-                total = rootNodes.Length,
-                items = rootNodes.Select(x => new RedirectRootNodeModel(x, _backOffice))
-            });
+	        var rootNodes = _backOffice.Settings.ContentApp.UserStartNodes
+		        ? _redirects.GetRootNodes(_backOffice.CurrentUser).ToArray()
```

Comment:
Isn't the `.ToArray()` call redundant here?

Reply:
Wait, that's out of date. Pull the latest in?

Reply:
My bad, caught it.